docs/moderator-handbook.md

```diff
@@ -249,80 +249,81 @@ Here's how moderators deal with violations of our [Code of Conduct](https://code
 
    - Someone has posted an advertisement and/or a self-promoting message/image (social media).
 
-   In all other situations - even situations where the code of conduct is violated - moderators should not delete the messages as they are important historic records. When you do delete a message, make sure you take a screenshot of it first! The screenshot can be logged in the #mod-log channel.
+   In all other situations - even situations where the [Code of Conduct](https://code-of-conduct.freecodecamp.org) is violated - moderators should not delete the messages as they are important historic records. When you do delete a message, make sure you take a screenshot of it first! The screenshot can be logged in the #mod-log channel.
 
    > [!NOTE]
-   > If the message contains material that would be illegal to take a screenshot of, copy the message link instead - provide that message link to @raisedadead to forward to Discord's Trust and Safety team.
+   > If the message contains material that would be illegal to take a screenshot of, copy the message link instead - provide that message link to **@raisedadead** to forward to Discord's Trust and Safety team.
 
-5. **Don’t use @all or @here**
+5. **Don’t use @all, @everyone, or @here**
 
-   Don’t use @all or @here under any circumstances! Every single person in that chat room will get a notification. In some cases, tens of
-   thousands of people.
+   Don’t use **@all**, **@everyone**, or **@here** under any circumstances! Every single person in that chat room will get a notification. In some cases, tens of thousands of people.
 
-   Instead, if you want people to see an announcement, you can pin it to the channel to allow everyone to read it.
+   Instead, if you want people to see an announcement, you can pin it to the channel to allow everyone to read it over time.
 
 6. **Don’t threaten to take action**
 
-   If a camper breaks the code of conduct, don’t threaten to take moderator action, and never warn them in public. Instead, talk to them privately using the bot's `private` command. No one else in that channel needs to know that you banned/suspended the person. If a violation was clearly unintended and doesn't warrant a suspension or private conversation, make the offending camper aware of his / her actions without making it come across as a warning. For example:
+   If a camper breaks the [Code of Conduct](https://code-of-conduct.freecodecamp.org), don’t threaten to take moderator action, and never warn them in public. Instead, talk to them privately using the bot's `private` command. No one else in that channel needs to know that you banned/suspended the person. If a violation was clearly unintended and doesn't warrant a suspension or private conversation, make the offending camper aware of his / her actions without making it come across as a warning.
+   
```

Comment:
```suggestion

```

Empty blankspace in this line needs to be removed.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix